spring-cloud-task-batch/src/test/java/org/springframework/cloud/task/batch/listener/TaskBatchExecutionListenerTests.java

```diff
@@ -91,6 +97,17 @@ public void testAutobuiltDataSource() {
 	}
 
 	@Test
+	public void testBeanFactoryReturnsSameInstance() throws Exception{
```

Comment:
The getObject method of the TaskBatchExecutionListenerFactoryBean in the SCT project.  And it was marked red as not covered by code coverage.